src/dom/Null.ts

```diff
@@ -41,10 +61,72 @@ export class Null extends DomValue(Object, IonTypes.NULL) {
         return true;
     }
 
+    // If a [type]Value() operation was called on this Null value, we need to see whether the associated Ion type
+    // supports that conversion. If it does, we'll return a JS null. If it doesn't, we'll throw an Error.
+    private _convertToJsNull(operation: string): null | never {
+        if (Null._operationIsSupported(this.getType(), operation)) {
+            return null;
+        }
+        throw new Error(`'${operation}' is not supported by Ion type ${this.getType().name}`);
```

Comment:
drop single quotes, add `()`?  as in:  `${operation}() is not supported...`

Reply:
Sure.